common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/rest/client/vrli/RestClientVrliV2.java

```diff
@@ -157,14 +158,10 @@ private void rewriteVcopsIntegrationInfo(AlertDTO alert) {
         ResourcesDTO resourceDto;
         try {
             resourceDto = restClientVrops.getResources();
-			logger.info(resourceDto.getResourceList().toString());
         } catch (Exception e) {
             throw new RuntimeException(
                     String.format("Unable to update vCOPs integration for alert '%s', unable to fetch vROPs resources: %s", alert.getName(), e.getMessage()));
         }
-        if (resourceDto == null) {
```

Comment:
From what I can tell this was removed since getResources will always return an object. Correct?

Reply:
Yes this is correct